ACE-update.pl

```diff
@@ -0,0 +1,549 @@
+#!/usr/bin/env perl
+
+# SYNTAX: ACE-update.pl [-b]
+# Use the -b switch when running from /data/acis-bak.
+
+# $Log$
+# Update History:
+# 16/10/02 - Added error trap for no arguments with usage example - JD
+# 25/03/03 - Some revisions to code - script no longer requires an argument
+#   path to load is obtained from /data/acis0/LoadReviews/current_load.txt
+# 07/05/03 - Change call from scp2 to scp.
+# 08/05/03 - Remove "-p" option in scp commands.
+# 10/29/03 - Added nadams to mailing
+# 02/18/04 - Added extra info to mailing and redirected mailing to acisdude
+# 03/23/04 - changed location of script "ephin_orbevents.pl"
+# 04/02/04 - added call to /data/acis0/LoadReviews/script/crm_saved_time.pl
+# 05/05/04 - updates for sw move to /data/acis & /data/acis-bak
+# 07/27/04 - updated location of ephin_orbevents.pl
+# 11/10/04 - converted to perl--NRAW
+# 12/09/09 - Actually crm_saved_time.pl is identical to the one in /data/acis/...... not acis0
+# 03/15/11 - Added a time check to make sure we don't set time backwards.
+# 06/23/15 - Copy ephin_orbevents file to acisweb machines.
+#
+# Update: October 5, 2018
+#         Gregg Germain
+#         - Add the copy of the orbevents file to acisway
+#
+# Update: February 14, 2019
+#         Gregg Germain
+#         - acisocc-w changed to acisway
+#
+# Update: June 20, 2019
+#              Gregg Germain
+#                - acisocc-v eliminated
+#
+# Update: April 4, 2022
+#              Gregg Germain
+#                - Added SCS-155 Deadman history files: SCS155-HIST.dat
+#                  and ACIS-SCS155-HIST.dat
+#
+# Update: August 24, 2022
+#              Gregg Germain
+#              - Removed file copies to luke-v and han-v
+#
+#--------------------------------------------------------------------
+#modules used
+#--------------------------------------------------------------------
+use Fcntl qw(:DEFAULT :flock);
+use File::Copy;
+use Mail::Mailer;
+use Getopt::Std;
+use Getopt::Long;
+use MachinePath(); #code to find the right directory
+
+
+#-----------------------------------
+#Check to be sure user is acisdude
+#------------------------------------
+$acisdude_uid=getpwnam acisdude;
+$acisdude_uid == $< or die "Error: Must be logged in as acisdude\n";
+
+#----------------------------------------
+#set variables
+#----------------------------------------
+$test=0;
+$test_dir="";
+# Recipient of email when ACS-update is run for score
+$recipient="acisdude\@head.cfa.harvard.edu";
+
+# Email recipient when ACS-update is run in test mode
+$test_email_recipient = "ggermain\@cfa.harvard.edu";
+
+$appx =`lr_suffix.pl`; # Null or "-bak"
+#$appx = "";
+
+if($appx) {
+    $isBackup = 1;
+}
+else{
+    $isBackup = 0;
+}
+$base_dir = </data/acis${appx}>;
+
+# FLU-MON directory definitions
+
+$han_flu=GetFluMon('han-v');
```

Comment:
Do you still want this defined?

Reply:
Commented out. Might want han-v to be defined in the future.